README.md

```diff
@@ -53,6 +53,17 @@ and health-checking mechanisms.
   ```
 
 * Install puppet-agent (&ge; 6.13.0)
+  
+  Check facter version details, if already installed / exist in system
+  ```sh
+  facter -v
+  ls -ll /usr/bin/facter
+  ```
+  Supported version is **facter >= 3.14.8**, If the facter version is **< 3.14.8** then follow the steps below: 
+  ```sh
+  yum erase -y $(rpm -q --whatprovides /usr/bin/facter) || rm -fv /usr/bin/facter
```

Comment:
As my experience today shows, it is safer to use this modified command:
```suggestion
  yum erase -y $(rpm -q --whatprovides $(readlink -f /usr/bin/facter)) || rm -fv /usr/bin/facter
```
Reason: in my case the RPM didn't know about `/usr/bin/facter` although the file was there and it was deployed or created by an RPM. Perhaps that symlink was created in `%post` section of the RPM, that's why we need to try resolving the symlink.